src/box/box.cc

```diff
@@ -867,6 +868,9 @@ box_check_small_alloc_options(void)
 	if (granularity < 4 || !is_exp_of_two(granularity))
 		tnt_raise(ClientError, ER_CFG, "granularity",
 			  "must be greater than 4 and exponent of two");
+	if (alloc_factor > 2.0 || alloc_factor <= 1.0)
+		tnt_raise(ClientError, ER_CFG, "slab_alloc_factor",
+			  "must be less than or equal to 2.0 and greater than 1.0");
```

Comment:
This is incompatible change so it should be reflected somehow in docs/changelogs